style.css

```diff
@@ -68,6 +92,53 @@ body {
   margin-bottom: 12px;
 }
 
+.contact-paragraph {
+  font-size: 20px;
+  text-align: center;
+  color: #ebebff;
+  line-height: 28px;
+  margin-bottom: 60px;
+}
+
+.input {
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  width: 327px;
+  padding: 15px;
+  margin-bottom: 20px;
+  color: #172b4d;
+  font-weight: 600;
```

Comment:
[OPTIONAL]
- [ ] The form implements flexbox correctly 👏🏽 . However, it's also important for its elements to adhere to the font style and weight from Figma as much as possible. It'll improve user experience as well.